### PR TITLE
Skip deploy on fork PRs

### PR DESCRIPTION
**Pull request by VIKTORVAV99**
## Description

Skips the preview deploy on forks.

### Double check

- [x] I have run `pnpx prettier --write .` and `poetry run format` to format my changes.

### CODE REVIEW

**Comment by VIKTORVAV99** [comment]
Don't worry about the unverified commits, I spun up a Codespace to create the PR from a fork and apparently GitHub don't automatically sign the commits from the Codespaces (I thought they would).

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -93,8 +93,10 @@ jobs:
     name: Deploy preview
     needs: [eslint, typecheck, setup, validate_generate_files, unit_test]
     if: >-
-      needs.setup.outputs.frontend != 'false' ||
-      needs.setup.outputs.workflows != 'false'
+      (needs.setup.outputs.frontend != 'false' ||
+      needs.setup.outputs.workflows != 'false') &&
+      github.event.pull_request.head.repo.full_name == github.repository &&
+      github.ref_name != 'master'
```
Comment (madsnedergaard) [inline .github/workflows/CI.yml:99]: What does the `ref_name != master` do - would it still run for PRs towards non-master branches?

Comment (VIKTORVAV99) [reply .github/workflows/CI.yml:99]: It skips the deployment on master as we have a separate job for that (that only runs on master).

**Review by madsnedergaard — APPROVED** [review]
LGTM, thanks for fixing it!

**Pull request merged**